From fe7fc969c1aca186b543fc6f0ec582f7780b8d54 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Tue, 30 Apr 2024 14:03:29 +0300
Subject: [PATCH] [SW-15774] build: remove dead code

As agents move out of the zephyr, there is no need for any agent related
code anymore

Change-Id: Ia4013989a23409f417413d75f8252c5c3570d9d6
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/package/zephyr/zephyr.mk b/package/zephyr/zephyr.mk
index d544a60..7246709 100644
--- a/package/zephyr/zephyr.mk
+++ b/package/zephyr/zephyr.mk
@@ -135,34 +135,6 @@
 ZEPHYR_COMPONENTS_LIST += agent-fwl
 endif
 
-ifdef BR2_arcle
-ZEPHYR_MARGIN_TOOL_AGENT_BOARD = hl_$(DEVICE_NAME)_agent_$(HL_TARGET)
-else
-ZEPHYR_MARGIN_TOOL_AGENT_BOARD = hl_$(DEVICE_NAME)_$(HL_TARGET)
-endif
-
-# first argument is agent name, second argument is additional configuration opts
-define CREATE_ZEPHYR_MARGIN_TOOL_AGENT
-$(eval AGENT_LOWER_CASE = $(strip $1)) \
-$(eval ZEPHYR_COMPONENTS_LIST += $(AGENT_LOWER_CASE)) \
-$(eval AGENT_UPPER_CASE = $(shell echo $(AGENT_LOWER_CASE) | tr a-z A-Z)) \
-$(eval AGENT_UPPER_CASE = $(strip $(AGENT_UPPER_CASE))) \
-$(eval ZEPHYR_AGENT_DIR = build-agent-$1) \
-$(eval ZEPHYR_$(AGENT_UPPER_CASE)_DIR = build-agent-$(AGENT_LOWER_CASE)) \
-$(eval ZEPHYR_$(AGENT_UPPER_CASE)_NAME = zephyr_agent_$(AGENT_LOWER_CASE)) \
-$(eval ZEPHYR_CONF_DEFAULT ?= $(ZEPHYR_AGENT_DIR)) \
-$(eval ZEPHYR_CONF_OPTS_AGENT = -DBOARD=$(ZEPHYR_MARGIN_TOOL_AGENT_BOARD)) \
-$(eval ZEPHYR_CONF_OPTS_AGENT += -DDEVICE=$(DEVICE_NAME)) \
-$(eval ZEPHYR_$(AGENT_UPPER_CASE)_SRCDIR = $(EMBEDDED_DIR)/zephyr-plugins/margin-tools/$(AGENT_LOWER_CASE)) \
-$(eval ZEPHYR_CONF_OPTS_AGENT += $(ZEPHYR_$(AGENT_UPPER_CASE)_SRCDIR)) \
-$(eval ZEPHYR_CONF_OPTS_AGENT += $2) \
-$(eval ZEPHYR_CONFIGURE_CMDS += && $$(ZEPHYR_CONF_ENV) $(HOST_DIR)/bin/cmake $$(ZEPHYR_CONF_OPTS) $(ZEPHYR_CONF_OPTS_AGENT) -B $(ZEPHYR_AGENT_DIR)) \
-$(eval ZEPHYR_BUILD_CMDS += && $(TARGET_MAKE_ENV) $(MAKE) -C $$(@D)/$(ZEPHYR_AGENT_DIR)) \
-$(eval ZEPHYR_INSTALL_TARGET_CMDS += && $(INSTALL) -D -m 0644 $$(@D)/$(ZEPHYR_AGENT_DIR)/zephyr/zephyr.bin $(STAGE_DIR)/zephyr_agent_$1.bin) \
-$(eval ZEPHYR_INSTALL_TARGET_CMDS += && $(INSTALL) -D -m 0644 $$(@D)/$(ZEPHYR_AGENT_DIR)/zephyr/zephyr.stat $(STATS_DIR)/zephyr_agent_$1.stat) \
-$(foreach module,$(ZEPHYR_$(AGENT_UPPER_CASE)_MODULES),$(call DEFINE_MODULE_ENVVARS,$(module),$(ZEPHYR_$(AGENT_UPPER_CASE)_DIR), $(ZEPHYR_$(AGENT_UPPER_CASE)_NAME)))
-endef
-
 ifdef BR2_PACKAGE_AGENT_HBM
 ZEPHYR_COMPONENTS_LIST += agent-hbm
 endif
